bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/BigtableWhileMatchResultScannerAdapter.java

```diff
@@ -69,6 +69,14 @@ public void close() {
           throw Throwables.propagate(ioe);
         }
       }
+
+      /**
+       * This is an HBase concept that was added in hbase 1.0.2.  It's not relevent for Cloud
+       * Bigtable.  It will not be called from the hbase code and should not be called by the user.
+       */
+      public boolean renewLease() {
+        return false;
```

Comment:
I am not sure if this is the right implementation.

If we want this to work with existing hbase code, we need an implementation that corresponds to Bigtable behavior.

If Bigtable always allow scan lease to be renewed, we should always return true.

If we do not allow user to call this method, we should throw an exception.